Podcast/EpisodeDetailViewController.swift

```diff
@@ -75,13 +75,19 @@ class EpisodeDetailViewController: ViewController, EpisodeDetailHeaderViewDelega
     func episodeDetailHeaderDidPressMoreButton(view: EpisodeDetailHeaderView) {
         guard let episode = episode else { return }
         let option1 = ActionSheetOption(type: .download(selected: episode.isDownloaded), action: nil)
+        let shareEpisodeOption = ActionSheetOption(type: .shareEpisode, action: {
```

Comment:
this code looks like it's coming up a lot, you could refactor this into a specific `ActionSheetOption` but i know a lot of the action sheet options could be refactored so it might be easier to do them all at once